package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "tdd-strict-equal",
+  "version": "1.0.0",
+  "description": "In this exercise you will be using a test driven development (TDD) approach to implement a `strictEqual()` function. The `strictEqual()` function will work the way the `===` operator does but you are not allowed to use this operator. For example, `strictEqual(5, '5')` would return `false` but `strictEqual('a', 'a')` would return `true`.",
+  "main": ".eslintrc.js",
+  "scripts": {
+    "lint": "./node_modules/.bin/eslint --format codeframe .",
+    "test": "./node_modules/.bin/mocha ./tests/*.test.js"
```

Comment:
Don't forget to add this line in your scripts if it is not already there

"lint:fix": "./node_modules/.bin/eslint --fix --format codeframe .",